cmd/build/helmify/static/README.md

```diff
@@ -0,0 +1,30 @@
+# Gatekeeper Helm Chart
+
+## Parameters
+
+| Parameter                 | Description                                                 | Default                                                                    |
+|:--------------------------|:------------------------------------------------------------|:---------------------------------------------------------------------------|
+| auditInterval             | The frequency with which audit is run                       | `60`                                                                       |
+| constraintViolationsLimit | The maximum # of audit violations reported on a constraint  | `20`                                                                       |
+| auditFromCache            | Take the roster of resources to audit from the OPA cache    | `false`                                                                    |
+| logLevel                  | Minimum log level                                           | `INFO`                                                                     |
+| image.pullPolicy          | The image pull policy                                       | `IfNotPresent`                                                             |
+| image.repository          | The Docker image repository                                 | `quay.io/open-policy-agent/gatekeeper`                                     |
+| image.release             | The image release tag to use                                | `v3.1.0-beta.8`                                                            |
```

Comment:
is `image.release` going to be a manual update?

Reply:
I can probably update the release makefile to fix this. Lemme take a look

Reply:
Updated the README/Makefile so that the version gets auto-updated.

It won't keep the lines nice and clean if the width changes, but should still render.